networking/ingress/getting-traffic-cluster.adoc

```diff
@@ -0,0 +1,101 @@
+[id="getting-traffic-cluster"]
+= Getting traffic into a cluster
+include::modules/common-attributes.adoc[]
+:context: getting-traffic-cluster
+
+toc::[]
+
+{product-title} provides multiple methods for communicating from
+outside the cluster with services running in the cluster.
+
+[NOTE]
+====
+The procedures in this section require prerequisites performed by the cluster
+administrator.
+====
+
+Administrators can expose a service endpoint that external traffic can reach, by
+assigning a unique external IP address to that service from a range of external
+IP addresses. Administrators can designate a range of addresses using a CIDR
+notation, which allows an application user to make a request against the cluster
+for an external IP address.
```

Comment:
The second sentence is not true in v4—`externalIPNetworkCIDRs` is not configurable.